src/main/webapp/resources/js/pages/projects/list/ProjectsTable.jsx

```diff
@@ -0,0 +1,224 @@
+import React, { useEffect, useReducer, useState } from "react";
+import {
+  Button,
+  Col,
+  Dropdown,
+  Icon,
+  Input,
+  Menu,
+  Row,
+  Table,
+  Typography
+} from "antd";
+import { getPagedProjectsForUser } from "../../../apis/projects/projects";
+import { PageWrapper } from "../../../components/page/PageWrapper";
+import { getI18N } from "../../../utilities/i18n-utilties";
+import {
+  dateColumnFormat,
+  idColumnFormat,
+  nameColumnFormat
+} from "../../../components/ant.design/table-renderers";
+
+const { Text } = Typography;
+
+const initialState = {
+  loading: true, // true when table fetching data
+  search: "", // Search query
+  current: 1, // Current page in pagination
+  pageSize: 10, // Current table size
+  total: undefined, // Total number of elements in the table
+  order: "descend", // Sort direction
+  field: "modifiedDate" // Sort field
+};
+
+const TYPES = {
+  SEARCH: "PROJECTS/SEARCH",
+  TABLE_CHANGE: "PROJECTS/TABLE_CHANGE"
+};
+
+const reducer = (state, action) => {
+  switch (action.type) {
+    case TYPES.TABLE_CHANGE:
+      /*
+      Case when the user has changed page or sort
+       */
+      return {
+        ...state,
+        ...action.payload
+      };
+    case TYPES.SEARCH:
+      /*
+      Case when the user is filtered the table
+       */
+      return {
+        ...state,
+        search: action.payload.search
+      };
+    default:
+      throw new Error(`No action found for type: ${action.type}`);
+  }
+};
+
+/**
+ * Component to display a
+ * @returns {*}
+ * @constructor
+ */
+export function ProjectsTable() {
+  const [state, dispatch] = useReducer(reducer, initialState);
+  const [total, setTotal] = useState(undefined);
+  const [loading, setLoading] = useState(false);
+  const [projects, setProjects] = useState(undefined);
+
+  useEffect(() => {
+    setLoading(true);
+    const params = {
+      current: state.current - 1, // Offset since table starts on page 1
+      pageSize: state.pageSize,
+      sortField: state.field,
+      sortDirection: state.order,
+      search: state.search
+    };
+    getPagedProjectsForUser(params).then(data => {
+      setProjects(data.projects);
+      setTotal(data.total);
+      setLoading(false);
+    });
+  }, [state]);
+
+  const handleTableChange = (pagination, filters, sorter) => {
+    const { pageSize, current } = pagination;
+    const { order, field } = sorter;
+    dispatch({
+      type: TYPES.TABLE_CHANGE,
+      payload: {
+        pageSize,
+        current,
+        order: order || "descend",
+        field: field || "modifiedDate"
+      }
+    });
+  };
+
+  const onSearch = value =>
+    dispatch({
+      type: TYPES.SEARCH,
+      payload: {
+        search: value,
+        currnet: 0
```

Comment:
Typo here. Should be `current: 0`

Reply:
Fixed in abd3b66b7d8f7829b2a4ec652971d8ab470c8774.  It actually needs to be `current: 1`, since the UI table uses 1 as the initial page, this is corrected for when the server call is actually made.